app/javascript/mastodon/features/emoji/unicode_to_filename.js

```diff
@@ -3,7 +3,7 @@
 
 // taken from:
 // https://github.com/twitter/twemoji/blob/47732c7/twemoji-generator.js#L848-L866
-exports.unicodeToFilename = (str) => {
+export const unicodeToFilename = (str) => {
```

Comment:
This change is unrelated, please remove your first commit from your PR. The issue you are trying to fix here has been fixed in `main`, so a rebase should fix this.